### PR TITLE
Doha - Lambda function added using terraform

### PR DESCRIPTION
**Pull request by DewDropsDoha**
Fixes #174  #192

- [ ] Does not depend on other PRs
- [x] Depends on other PR being merged first: #197 

## What kind of change does this PR introduce (check at least one)

- [ ] Bugfix
- [ ] Improvement
- [x] New feature
- [ ] Refactor
- [ ] Infrastructure
- [ ] Other (please specify)

## What was done
Lambda initialized using terraform. 
Refs: https://learn.hashicorp.com/tutorials/terraform/lambda-api-gateway?in=terraform/aws#create-and-upload-lambda-function-archive

## How to test

- [ ] Needs regression testing (add QA needed label)

## Deploy risk & follow up

Please assess risk & follow up on deployment of this PR. Elaborate your choice if selecting anything other than 'No risk'.

- [x] No risk
- [ ] Low
- [ ] High

QA - if regression testing was requested:

- [ ] QA has verified

### CODE REVIEW

File: doha-aws-node-project/terraform/terraform.tfstate [hunk 1]
```diff
@@ -1,8 +1,301 @@
 {
```
[the remark this answers is not in the document]
Comment (DewDropsDoha) [reply doha-aws-node-project/terraform/terraform.tfstate:1]: tfstate files added in gitignore. Commit 34d87b3

File: doha-aws-node-project/terraform/terraform.tfstate.backup [hunk 2]
```diff
@@ -0,0 +1,273 @@
+{
```
[the remark this answers is not in the document]
Comment (DewDropsDoha) [reply doha-aws-node-project/terraform/terraform.tfstate.backup:1]: tfstate files added in gitignore. Commit 34d87b3

File: doha-aws-node-project/terraform/terraform.tfstate (hunk 1 above)
[the remark this answers is not in the document]
Comment (mhazad) [reply doha-aws-node-project/terraform/terraform.tfstate:1]: And then you need to delete the file from git.

File: doha-aws-node-project/terraform/terraform.tfstate.backup (hunk 2 above)
[the remark this answers is not in the document]
Comment (mhazad) [reply doha-aws-node-project/terraform/terraform.tfstate.backup:1]: And then you need to delete the file from git.

**Review by mhazad — CHANGES_REQUESTED** [review]
Pls, remove unnecessary files.

File: doha-aws-node-project/terraform/terraform.tfstate (hunk 1 above)
[the remark this answers is not in the document]
Comment (DewDropsDoha) [reply doha-aws-node-project/terraform/terraform.tfstate]: tfstate files removed. Commit 3df427f

File: doha-aws-node-project/terraform/terraform.tfstate.backup (hunk 2 above)
[the remark this answers is not in the document]
Comment (DewDropsDoha) [reply doha-aws-node-project/terraform/terraform.tfstate.backup]: tfstate files removed. Commit 3df427f

**Review by mhazad — APPROVED** [review]
LGTM

**Pull request merged**